include/mixer.h

```diff
@@ -98,6 +99,8 @@ enum LINE_INDEX : uint8_t {
 	// standard at the host-level, then additional line indexes would go here.
 };
 
+enum class ChannelFeature { Stereo, ReverbSend, ChorusSend };
```

Comment:
This is a real pleasure to read throughout the code :+1: 

When I see:
``` c++
 "INNOVATION",
   {ChannelFeature::ReverbSend,
     ChannelFeature::ChorusSend});
```

I know exactly what we're turning on.

The bool approach just doesn't cut it anymore! :smiley: 

``` c++
( ... , "INNOVATION", true, false, true)
```

Reply:
More than a single bool function arg is definitely a code-smell.